{{ cookiecutter.project_slug }}/.github/workflows/ci.yml

```diff
@@ -12,7 +12,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [windows-latest, ubuntu-latest, macos-latest]
+        os:
+          - ubuntu-latest
+{%- if cookiecutter.use_windows_ci_runner %}
+          - windows-latest
+{%- endif %}
+{%- if cookiecutter.use_macos_ci_runner %}
+          - macos-latest{% endif %}
```

Comment:
This results in something like this
```
        os:
          - ubuntu-latest
          - windows-latest
          - macos-latest
```

Is that equivalent to `os: [windows-latest, ubuntu-latest, macos-latest]`?

My pedantic self wants there to be a newline before that last `{% endif %}`

Reply:
Yep, it's just different syntax. It seemed more finicky to do it the other way (though there probably is a way to make `jinja` do it).

Getting the newlines right was such a faff :laughing:. If you get them wrong the tests won't pass because the formatting will be off.